modules/ROOT/pages/stats-monitoring.adoc

```diff
@@ -232,291 +315,657 @@ Stats for each database declared in the config file
 
 Stats relative to a database declared in the config file.
 
-==== cache
+=== cache
 
 Stats relative to caching
 
-===== abandoned_seqs
-
+==== abandoned_seqs
+Description::
 The number of skipped sequences that were not found after 60 minutes and were abandoned.
 
-===== chan_cache_active_revs
-
+==== chan_cache_active_revs
+Description::
 The number of active revisions in the channel cache.
 
-===== chan_cache_hits
+==== chan_cache_bypass_count
+Description::
+Number of transient bypass channel caches created to serve requests when the channel cache was at capacity.
 
-Channel cache requests fully served by the cache.
+==== chan_cache_channels_added
+Description::
+Number of channel caches added.
+Constraints::
+Doesn't decrease when the channel is removed.
+Similar to <<chan_cache_num_channels>> but doesn't track removals.
 
-Channel Cache Hit Ratio = `chan_cache_hits` / (`chan_cache_hits` + `chan_cache_misses`)
+==== chan_cache_channels_evicted_inactive
+Description::
+Number of channel cache channels evicted due to inactivity.
 
-===== chan_cache_max_entries
+==== chan_cache_channels_evicted_nru
+Description::
+Number of active channel cache channels evicted, based on ‘not recently used’ criteria.
 
-Size of the largest channel cache.
+==== chan_cache_compact_count
+Description::
+Number of channel cache compaction runs
 
-Helps with channel cache tuning, and as a hint on cache size variation (when compared to average cache size).
+==== chan_cache_compact_time
+Description::
+Amount of time channel cache compaction has taken in total, across all compaction runs.
 
-===== chan_cache_misses
+==== chan_cache_hits
+Description::
+Channel cache requests fully served by the cache.
+Use Case::
```

Comment:
Make capitalisation consistent throughout. (`Use case` vs. `Use Case`)

Reply:
Changed to Use Case throughout